tests/.pest/snapshots/Countries/IranTest/it_can_calculate_iran_holidays.snap

```diff
@@ -0,0 +1,30 @@
+[
+    {
+        "name": "Islamic Revolution Day",
+        "date": "2024-02-11"
+    },
+    {
+        "name": "National Oil Industry Day",
+        "date": "2024-03-20"
+    },
+    {
+        "name": "Nourouz",
+        "date": "2024-03-23"
```

Comment:
we have 4 days of holiday for Nowruz and you wrote the same code but Here it says something else!